src/scripts/main.js

```diff
@@ -1,3 +1,73 @@
 'use strict';
 
-// write code here
+const addRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const addColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+const table = document.querySelector('table');
+let rows = document.querySelectorAll('tr');
+const max = 10;
+const min = 2;
+let rowsNumber = rows.length;
+let columnsNumber = rows[0].children.length;
+
+addRow.addEventListener('click', () => {
+  const row = rows[0].cloneNode(true);
+
+  table.append(row);
+  rows = document.querySelectorAll('tr');
+  rowsNumber++;
+
+  if (rowsNumber === max) {
+    addRow.disabled = true;
+  }
+
+  if (rowsNumber > min) {
+    removeRow.disabled = false;
+  }
+});
+
+removeRow.addEventListener('click', () => {
+  table.lastChild.remove();
```

Comment:
```suggestion
  table.lastElementChild.remove();
```
work wiht lastElementChild
lastChild not always element. it may be some text or something else